lms/djangoapps/sga_api/views.py

```diff
@@ -147,12 +147,15 @@ def post(self, request):
                 data={"error_message": "Action must be enroll or unenroll"},
                 status=status.HTTP_400_BAD_REQUEST
             )
+        if type(email_students) is not bool:
```

Comment:
I would use `isinstance` method instead of `type`

Reply:
I agree it is better. I changed the code.